pdfjs-web/pdf_page_view.js

```diff
@@ -223,7 +223,7 @@ var PDFPageView = (function PDFPageViewClosure() {
           this.zoomLayer = this.canvas.parentNode;
         }
       }
-      if (this.zoomLayer) {
+      if (this.zoomLayer && this.zoomLayer.firstChild) {
```

Comment:
I would prefer to avoid modifying third-party code unless we have a really good reason for that. However, admittedly `canvas` element is removed from `PDFPageView.reset` method after zom in while `zoomLayer` is kept alive, no without its firstChild.  It is not obvious what we're doing wrong.

I looked at the latest PDFJs sources and it seems that manipulations with `canvasWrapper` were removed from PDFPageView.reset method. Maybe we can just upgrade to more recent version of PDFJS?

Reply:
Another option is to just add empty `canvas` elements to those pages which are outside of the visible area, so that zooming didn't break.

Reply:
In process of updating libs